configure.ac

```diff
@@ -1473,8 +1473,8 @@ AC_CHECK_HEADERS([errno.h \
 		  sys/socket.h \
 		  sys/statvfs.h \
 		  sys/syscall.h \
-		  sys/sysctl.h \
 		  sys/syslimits.h \
+		  sys/sysctl.h \
```

Comment:
I don't think this change actually does anything?
So let's keep the headers in alphabetical order.

Reply:
+1